### ISSUE TITLE
`make build-deps` does not install `autofonce_lib`

### ISSUE THREAD

**Issue by GitMensch** [OCamlPro/superbol-studio-oss#79]
which is required to run `make superbol-free`

**Comment by nberth** [comment]
Actually, I think the issue is that `make superbol-free` should not require `autofonce_lib`.

**Comment by GitMensch** [comment]
For completeness:

```sh
 make
./scripts/before.sh build
opam exec -- dune build @install
Read 14 sample input sentences and 14 error messages.
File "src/testing/superbol_testutils/dune", line 8, characters 13-26:
8 |   (libraries autofonce_lib )
                 ^^^^^^^^^^^^^
Error: Library "autofonce_lib" not found.
-> required by library "superbol_testutils" in
   _build/default/src/testing/superbol_testutils
-> required by _build/default/META.superbol_testutils
-> required by _build/install/default/lib/superbol_testutils/META
-> required by _build/default/superbol_testutils.install
-> required by alias install
;; linking.sexp                                      
;; generated by linking_flags.sh
()
make: *** [Makefile:22: build] Error 1         
```

**Comment by nberth** [comment]
Should be addressed now that #105 is merged.
